.gitlab-ci.yml

```diff
@@ -481,7 +474,7 @@ filesystem.sh:
     - RUNNER:
       # don't execute on fedora-39 yet until https://lkml.kernel.org/linux-xfs/20230823220225.3591135-1-david@fromorbit.com/#t is in the kernel
```

Comment:
I guess this comment can be removed now :)

Reply:
removed, and switched to 40 since it seemed to fail still